src/main.rs

```diff
@@ -109,6 +172,33 @@ fn main() -> Result<()> {
     let mut buffer = Vec::new();
     input.read_to_end(&mut buffer)?;
 
+    let schema_string_read_result = opts.read_schema_to_string();
+    let query_string_read_result = opts.read_query_to_string();
+    let input_json: serde_json::Value = serde_json::from_slice(&buffer)?;
+    let mut scale_factor = 1.0;
+
+    if let (Ok(schema_string), Ok(query_string)) =
+        (schema_string_read_result, query_string_read_result)
+    {
```

Comment:
Will the user be notified if they pass a path for the schema or the query file and the file doesn't exist at the specified paths? Also should anything be emitted if one flag is passed but not the other?